director/director_api.go

```diff
@@ -154,3 +158,73 @@ func ConfigFilterdServers() {
 		filteredServers[sn] = permFiltered
 	}
 }
+
+// Start a goroutine to query director's Prometheus endpoint for origin/cache server I/O stats
+// and save the value to the corresponding serverAd
+func ConfigServerIOQuery(ctx context.Context, egrp *errgroup.Group) {
```

Comment:
I have been trying to use `Launch` in the function name to indicate a goroutine will be created by a given routine.
```suggestion
func LaunchServerIOQuery(ctx context.Context, egrp *errgroup.Group) {
```

In general, this function is hard to read due to the level of indentation.  Instead of having an anonymous function inside `egrp.Go`, can you define the function in the body of `LaunchServerIOQuery` and then later call `egrp.Go`?